geometry_msgs/Pose2D goal
std_msgs/String id
---
vishnu_rudrasamudram_intern/Path path